sql/core/src/test/scala/org/apache/spark/sql/internal/ExecutorSideSQLConfSuite.scala

```diff
@@ -59,7 +59,8 @@ class ExecutorSideSQLConfSuite extends SparkFunSuite with SQLTestUtils {
         val pathString = path.getCanonicalPath
         spark.range(10).select('id.as("ID")).write.json(pathString)
         spark.range(10).write.mode("append").json(pathString)
-        assert(spark.read.json(pathString).columns.toSet == Set("id", "ID"))
+        val df = spark.read.option("samplingRatio", 1).json(pathString)
```

Comment:
oh, I just found `samplingRatio` is 1.0 by default. The flaky test should not because of this.